source/common/network/utility.cc

```diff
@@ -542,11 +539,71 @@ void passPayloadToProcessor(uint64_t bytes_read, Buffer::RawSlice& slice,
                                      std::move(buffer), receive_time);
 }
 
+Api::IoCallUint64Result receiveMessage(uint64_t max_packet_size, Buffer::InstancePtr& buffer,
```

Comment:
This should be in an anonymous namespace.

Reply:
Moved `receiveMessage(...)` into anonymous namespace.